/tmp/glance/doc/source/conf.py:188:25: DUO106 insecure use of "os" module
/tmp/glance/glance/common/crypt.py:25:1: DUO133 use of "Crypto" module is insecure
/tmp/glance/glance/common/crypt.py:26:1: DUO133 use of "Crypto" module is insecure
/tmp/glance/glance/common/crypt.py:27:1: DUO133 use of "Crypto" module is insecure
/tmp/glance/glance/common/crypt.py:44:14: DUO102 insecure use of "random" module, prefer "random.SystemRandom"
/tmp/glance/glance/store/filesystem.py:211:20: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/test_bin_glance_cache_manage.py:62:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/test_cache_middleware.py:71:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/test_cache_middleware.py:255:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/v1/test_copy_to_file.py:126:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/v1/test_copy_to_file.py:127:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/v1/test_copy_to_file.py:148:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/v1/test_copy_to_file.py:149:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/v1/test_copy_to_file.py:237:30: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/v1/test_copy_to_file.py:238:30: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/v1/test_misc.py:65:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/v1/test_s3.py:97:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/v1/test_s3.py:220:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/v1/test_s3.py:221:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/v1/test_s3.py:242:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/v1/test_s3.py:243:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/v1/test_s3.py:311:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/v1/test_s3.py:312:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/v1/test_ssl.py:160:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/v1/test_ssl.py:212:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/v1/test_ssl.py:213:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/v1/test_ssl.py:429:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/v1/test_ssl.py:452:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/unit/test_filesystem_store.py:89:29: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/unit/test_image_cache.py:378:15: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/unit/test_image_cache.py:422:39: DUO102 insecure use of "random" module, prefer "random.SystemRandom"
/tmp/glance/glance/tests/unit/test_image_cache.py:480:39: DUO102 insecure use of "random" module, prefer "random.SystemRandom"
/tmp/glance/glance/tests/unit/test_s3_store.py:76:24: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/unit/test_s3_store.py:240:29: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/unit/test_s3_store.py:286:33: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/unit/v1/test_api.py:2865:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/unit/v1/test_api.py:2889:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/unit/v1/test_api.py:2921:24: DUO130 insecure use of "hashlib" module
